sway/commands/input/xkb_switch_layout.c

```diff
@@ -13,6 +13,26 @@ static void switch_layout(struct wlr_keyboard *kbd, xkb_layout_index_t idx) {
 		kbd->modifiers.latched, kbd->modifiers.locked, idx);
 }
 
+static xkb_layout_index_t get_current_layout_index(struct wlr_keyboard *kbd) {
+	xkb_layout_index_t num_layouts = xkb_keymap_num_layouts(kbd->keymap);
```

Comment:
I'd like to add `assert(num_layouts > 0)` here to ensure we don't return garbage from this function.